app/views/pages/index.html.erb

```diff
@@ -1,4 +1,23 @@
 <div class="container text-center">
   <h1>Pages#index</h1>
   <p>Find me in app/views/pages/index.html.erb</p>
+  <% if current_user %>
```

Comment:
This will work when the pull request gets merged with the previous one, yeah. As it is it will give an error.